nntrainer/models/neuralnet.cpp

```diff
@@ -911,7 +903,7 @@ void NeuralNetwork::print(std::ostream &out, unsigned int flags,
   }
 
   // TODO: get sorted layers if initialized
-  auto layers = model_graph.getLayers();
+  auto layers = model_graph.getLayerNodes();
```

Comment:
Not directly related, I wonder if this can return reference, or sonner or later if this function gonna be deprecated

Reply:
getting all nodes (be it sorted or unsorted) will be deprecated. Only iterators will be allowed to get, and the caller can iterate over the graph (there will be graph.cbegin()/cend() and graph.scbegin()/scend() -> s corresponds to sorted)